Source/XCDBLD/XcodeVersion.swift

```diff
@@ -42,4 +42,8 @@ public struct XcodeVersion {
 			}
 			.single()?.value
 	}
+
+	public static func makeString() -> String {
```

Comment:
I would rather return and error here. Was there any practical reason for this silently defaulting to Unknown?

Reply:
The defaulting was in https://github.com/Carthage/Carthage/blob/master/Source/CarthageKit/Project.swift#L131
but I need it in Xcode.swift, so I moved it to XcodeVersion.swift

Reply:
Precisely at:  https://github.com/openium/Carthage/blob/master/Source/CarthageKit/Xcode.swift#L14

Reply:
ok, cool